test/targets/create-target.e2e-spec.ts

```diff
@@ -77,8 +78,21 @@ describe('POST /targets', () => {
       })
 
       it('should return the newly created target', async () => {
-        const target = await targets.last()
-        expect(response.body).toEqual(TargetDto.from(target))
+        firstTarget = await targets.last()
+        expect(response.body.target).toEqual(TargetDto.from(firstTarget))
+      })
+      
+      describe('when there is another target nearby with the same topic', () => {
+        it('should create a new match', async () => {
+          response = await request(app.getHttpServer())
+            .post('/targets')
+            .set('Authorization', `Bearer ${accessToken}`)
+            .send({ ...mockTarget2, topicId: mockTopic.id })
+            .expect('Content-Type', /json/)
+            .expect(201)
+          firstTarget.topic = undefined
```

Comment:
What is this for?

Reply:
Matches don't include the topic since they will always have the same topic as the matching target. Maybe I should do this in some other way?

Reply:
Yeah, I see now that it was to match the response from the controller but it's not clear at first sight

Reply:
I agree it's not. Any suggestions?